src/modules/commands/discord/urbandictionary.ts

```diff
@@ -51,7 +57,7 @@ export default new Command<discord.Message>({
         return camelcaseKeys(definitions, { deep: true })
       })) as Definition[]
 
-    const re = /\[(\w+(\s\w+)*?)\]/g
+    const re = /\[|\]/g
```

Comment:
You don't need a special variable for this.

Reply:
I use it 4 times and I don't want to change it 4 times if I need to make a change there